src/main/kotlin/com/terraformation/backend/report/ReportService.kt

```diff
@@ -71,12 +75,16 @@ class ReportService(
       organizationId: OrganizationId,
       body: LatestReportBodyModel? = null
   ): LatestReportBodyModel {
+    val isAnnual = ZonedDateTime.now(clock).minusMonths(3).quarter == 4
```

Comment:
Since `fetchOneById` also calls `populateBody` might this return an incorrect result when accessing reports that are not being just created?

Reply:
Good catch. This will mess up if the report is in progress for an entire quarter.